subsys/testsuite/ztest/include/ztress.h

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef TESTSUITE_ZTEST_INCLUDE_ZTRESS_H__
+#define TESTSUITE_ZTEST_INCLUDE_ZTRESS_H__
+
+#include <sys/util.h>
+#include <kernel.h>
+
+/** @internal Internal ID's to distinguish context type. */
+#define ZTRESS_ID_THREAD 0
+#define ZTRESS_ID_K_TIMER 1
+
+/** @brief Declare k_timer handler execution context.
+ *
+ * Context is executed in the k_timer handler context which typically means
+ * interrupt context. This context will any other used in the set.
+ *
+ * @note There can only be one k_timer context in the set and it must be the
+ * first argument of @ref ZTRESS_EXECUTE.
```

Comment:
Point of clarification - Is the intent of this statement to convey that there must be exactly one or at most one handler in the set that executes within the k_timer context?